authorization/invitation/model/invitation_model_service.go

```diff
@@ -2,41 +2,51 @@ package model
 
 import (
 	"context"
-
 	"fmt"
+
 	"github.com/fabric8-services/fabric8-auth/account"
 	"github.com/fabric8-services/fabric8-auth/authorization"
 	"github.com/fabric8-services/fabric8-auth/authorization/invitation"
-	invRepo "github.com/fabric8-services/fabric8-auth/authorization/invitation/repository"
-	permissionService "github.com/fabric8-services/fabric8-auth/authorization/permission/model"
-	"github.com/fabric8-services/fabric8-auth/authorization/repository"
-	resourceRepo "github.com/fabric8-services/fabric8-auth/authorization/resource/repository"
+	invitationrepo "github.com/fabric8-services/fabric8-auth/authorization/invitation/repository"
+	permissionsrv "github.com/fabric8-services/fabric8-auth/authorization/permission/model"
+	resource "github.com/fabric8-services/fabric8-auth/authorization/resource/repository"
+	resourcerepo "github.com/fabric8-services/fabric8-auth/authorization/resource/repository"
+	role "github.com/fabric8-services/fabric8-auth/authorization/role/repository"
 	"github.com/fabric8-services/fabric8-auth/errors"
+
 	"github.com/satori/go.uuid"
 )
 
+// repositories interface lets us avoid an import cycle
+type repositories interface {
```

Comment:
Nice idea, I identified this as an alternative if import cycles became a serious problem.